server/routes/api/ads.js

```diff
@@ -134,4 +134,28 @@ router.post('/', async (req, res) => {
     }
 });
 
+router.post('/close/:id', async (req, res) => {
+    const { id } = req.params;
+
+    const { rows } = await db.query(`
+    UPDATE jobs.advertisements ads
+	    SET is_close=true
```

Comment:
The query identation is not properly done

Reply:
Fixed